Data/ByteString/Char8.hs

```diff
@@ -693,12 +693,38 @@ elemIndices = B.elemIndices . c2w
 -- returns the index of the first element in the ByteString satisfying the predicate.
 findIndex :: (Char -> Bool) -> ByteString -> Maybe Int
 findIndex f = B.findIndex (f . w2c)
-{-# INLINE findIndex #-}
+{-# INLINE[1] findIndex #-}
 
 -- | The 'findIndices' function extends 'findIndex', by returning the
 -- indices of all elements satisfying the predicate, in ascending order.
 findIndices :: (Char -> Bool) -> ByteString -> [Int]
 findIndices f = B.findIndices (f . w2c)
+{-# INLINE[1] findIndices #-}
```

Comment:
Space before `[1]` please.

Reply:
I fixed this but also noticed that there were `{-# INLINE[1] #-}` pragmas in `Data.ByteString.Builder.Internal` and `Data.ByteString.Builder.Prim`, in case it matters.